src/vorta/views/archive_tab.py

```diff
@@ -488,15 +492,28 @@ def list_result(self, result):
             self.populate_from_profile()
 
     def refresh_archive_info(self):
-        archive_name = self.selected_archive_name()
-        if archive_name is not None:
-            params = BorgInfoArchiveJob.prepare(self.profile(), archive_name)
-            if params['ok']:
-                job = BorgInfoArchiveJob(params['cmd'], params, self.profile().repo.id)
-                job.updated.connect(self._set_status)
-                job.result.connect(self.info_result)
-                self._toggle_all_buttons(False)
-                self.app.jobs_manager.add_job(job)
+        selected_archives = self.archiveTable.selectionModel().selectedRows()
+        profile = self.profile()
+
+        name_list = []
+        for index in selected_archives:
+            name_list.append(self.archiveTable.item(index.row(), 4).text())
+
+        archive_list = (
+            profile.repo.archives.select().where(ArchiveModel.name << name_list).order_by(ArchiveModel.time.desc())
+        )
+
+        archive_names = [archive.name for archive in archive_list]
+
+        for archive_name in archive_names:
+            if archive_name is not None:
+                params = BorgInfoArchiveJob.prepare(self.profile(), archive_name)
+                if params['ok']:
```

Comment:
Can you log the problem when `not params['ok']` or maybe even display it in the GUI.

Reply:
Similar to this:
https://github.com/borgbase/vorta/blob/2cb9afd4d551dfa5f251d6edafb29de020a4234e/src/vorta/views/archive_tab.py#L810-L813C19